app/views/posts/_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="form-post text-start my-5">
+  <div class="form">
+
+    <% post = @post %>
```

Comment:
Should pass as parameter to partials,
Should always avoid using Instance variables in partials.